components/public-api/gitpod/experimental/v1/editor_service.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+
+package gitpod.experimental.v1;
+
+option go_package = "github.com/gitpod-io/gitpod/components/public-api/go/experimental/v1";
+
+service EditorService {
+    rpc ListEditorOptions(ListEditorOptionsRequest) returns (ListEditorOptionsResponse) {}
+}
+
+message ListEditorOptionsRequest {}
+
+message ListEditorOptionsResponse {
+    repeated EditorOption options = 1;
+}
+
+message EditorOption {
+    // The unique identifier for an editor.
+    string id = 1;
+
+    // Human readable title text of the editor (plain text only).
+    string title = 2;
+
+    // The type of the editor, currently browser or desktop.
+    enum EditorType {
+      EDITOR_TYPE_UNSPECIFIED = 0;
+      EDITOR_TYPE_BROWSER = 1;
+      EDITOR_TYPE_DESKTOP = 2;
+    }
+    EditorType type = 3;
+
+    // The logo for the editor
+    string logo = 4;
+
+    // Text of an optional label next to the editor option like “Insiders” (plain
+    // text only).
+    string label = 5;
+
+    message Version {
+        // The semantic version of the stable editor image.
+        string stable = 1;
+
+        // The semantic version of the latest editor image.
+        string latest = 2;
+    }
+
+    Version version = 6;
```

Comment:
I won't do it like that we may have later other different properties for latest or stable. I would rather make it flat `option.latestVersion` or have `option.latest.version`

Reply:
Made it the latter:
```
  "stable": {
    "version": "1.0.0"
  },
  "latest": {
    "version": "1.0.0"
  }
```